concordium-consensus/src/Concordium/KonsensusV1/TreeState/Implementation.hs

```diff
@@ -636,20 +636,20 @@ commitTransaction ::
     m ()
 commitTransaction rnd bh ti transaction =
     transactionTable
-        . ttHashMap
+        . TT.ttHashMap
         . at' (getHash transaction)
         . traversed
         . _2
-        %=! addResult bh rnd ti
+        %=! TT.addResult bh rnd ti
 
 -- |Add a transaction to the transaction table if its nonce/sequence number is at least the next
 -- non-finalized nonce/sequence number. The return value is 'True' if and only if the transaction
 -- was added.
 -- When adding a transaction from a block, use the 'Round' of the block. Otherwise use round @0@.
 -- The transaction must not already be present.
-putTransaction :: (MonadState (SkovData pv) m) => Round -> BlockItem -> VerificationResult -> m Bool
-putTransaction rnd transaction verRes = do
-    added <- transactionTable %%=! addTransaction transaction (commitPoint rnd) verRes
+addTransaction :: (MonadState (SkovData pv) m) => Round -> BlockItem -> VerificationResult -> m Bool
+addTransaction rnd transaction verRes = do
```

Comment:
I agree partly, but on the other hand I don't think we should name our our functions (api) differently because of a dependency. I think we should re-evaluate this as part of https://github.com/Concordium/concordium-node/issues/768